mangaki/mangaki/utils/lasso.py

```diff
@@ -0,0 +1,63 @@
+from mangaki.utils.common import RecommendationAlgorithm
+from mangaki.settings import DATA_DIR
+from scipy.sparse import coo_matrix, load_npz
+from sklearn.linear_model import Lasso
+from sklearn.preprocessing import scale
+from collections import Counter, defaultdict
+from mangaki.utils.stats import avgstd
+import os.path
+
+
+def relu(x):
+    return max(-2, min(2, x))
+
+
+class MangakiLASSO(RecommendationAlgorithm):
+    def __init__(self, with_bias=True, alpha=0.01):
+        super().__init__()
+        self.alpha = alpha
+        self.with_bias = with_bias
+
+    def load_tags(self, T=None):
+        if T is None:
+            T = load_npz(os.path.join(DATA_DIR, 'balse/tag-matrix.npz')).tocsc()
+            T_scaled = scale(T, with_mean=False).toarray()
```

Comment:
First thing, I think we should add `copy=False` to benefit from CSC format.
Second thing, if you're not using the `mean` due to the sparsity destruction effect, just make it dense beforehand because nevertheless, you're making it dense afterwards :p.
Third thing, add an `else` clause when `T` is provided so that `T_scaled` exists in case of user-provided matrix. I think we don't want to scale in this case, because we assume the user will do it, let's add a comment to clarify this behavior.
Fourth thing, we may want a `should_scale` boolean parameter.